addons/mail/security/ir.model.access.csv

```diff
@@ -1,48 +1,58 @@
+access_mail_template,mail.template,model_mail_template,base.group_user,1,1,1,0
+access_mail_template_system,mail.template_system,model_mail_template,base.group_system,1,1,1,1
+access_mail_shortcode,mail.shortcode,model_mail_shortcode,base.group_user,1,1,1,1
+access_mail_shortcode_portal,mail.shortcode.portal,model_mail_shortcode,base.group_portal,1,0,0,0
+access_mail_activity_all,mail.activity.all,model_mail_activity,,0,0,0,0
+access_mail_activity_user,mail.activity.user,model_mail_activity,base.group_user,1,1,1,1
+access_mail_activity_type_all,mail.activity.type.all,model_mail_activity_type,,0,0,0,0
+access_mail_activity_type_user,mail.activity.type.user,model_mail_activity_type,base.group_user,1,0,0,0
+access_mail_activity_type_system,mail.activity.type.system,model_mail_activity_type,base.group_system,1,1,1,1
+access_mail_blacklist_system,access_mail_blacklist_system,model_mail_blacklist,base.group_system,1,1,1,1
+access_mail_wizard_invite,access.mail.wizard.invite,model_mail_wizard_invite,base.group_user,1,1,1,0
+access_mail_compose_message,access.mail.compose.message,model_mail_compose_message,base.group_user,1,1,1,0
+access_mail_compose_message_portal,access.mail.compose.message.portal,model_mail_compose_message,base.group_portal,1,1,1,0
+access_mail_resend_cancel,access.mail.resend.cancel,model_mail_resend_cancel,base.group_user,1,1,1,0
+access_mail_resend_message,access.mail.resend.message,model_mail_resend_message,base.group_user,1,1,1,0
+access_mail_resend_partner,access.mail.resend.partner,model_mail_resend_partner,base.group_user,1,1,1,0
+access_mail_template_preview,access.mail.template.preview,model_mail_template_preview,base.group_user,1,1,1,0
+access_mail_blacklist_remove_system,acesss.mail.blacklist.remove.system,model_mail_blacklist_remove,base.group_system,1,1,1,1
+access_mail_user_settings_all,mail.user.settings,model_mail_user_settings,,0,0,0,0
+access_mail_user_settings_user,mail.user.settings,model_mail_user_settings,base.group_user,1,1,1,1
+access_mail_volume_setting_all,mail.volume.setting,model_mail_volume_setting,,0,0,0,0
+access_mail_volume_setting_user,mail.volume.setting,model_mail_volume_setting,base.group_user,1,1,1,1
+ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_actions_report,base.group_user,1,0,0,0
+access_mail_ice_server_portal,mail.ice.server.portal,model_mail_ice_server,base.group_portal,0,0,0,0
+access_mail_ice_server_user,mail.ice.server.user,model_mail_ice_server,base.group_user,0,0,0,0
+access_mail_ice_server_system,mail.ice.server.system,model_mail_ice_server,base.group_system,1,1,1,1
+access_mail_rtc_session_all,mail.rtc.session.all,model_mail_rtc_session,,1,0,0,0
+access_mail_rtc_session_user,mail.rtc.session.user,model_mail_rtc_session,base.group_user,1,1,1,1
+access_mail_rtc_session_system,mail.rtc.session.system,model_mail_rtc_session,base.group_system,1,1,1,1
```

Comment:
That's redundant, if users have full access then system users also have full access by definition.